gradle/libs.versions.toml

```diff
@@ -0,0 +1,78 @@
+[versions]
+intellij-plugin = "1.17.2"
+license-plugin = "0.16.1"
+artifactory-plugin = "4.33.1"
+protobuf-plugin = "0.9.4"
+cyclonedx-plugin = "1.8.2"
+
+commons-lang3 = "3.14.0"
+findbugs-jsr305 = "3.0.2"
+junit-bom = "5.9.2"
+okhttp3-mockwebserver = "4.12.0"
+mockito-kotlin = "5.2.1"
+awaitility = "4.2.0"
+
+sonarqube = "4.4.1.3373"
+
+sonarlint-core = "9.8.0.76914"
+
+sonar-java = "7.30.1.34514"
+sonar-javascript = "10.11.0.25043"
+sonar-php = "3.33.0.11274"
+sonar-python = "4.14.0.14263"
+sonar-kotlin = "2.20.0.4382"
+sonar-slang = "1.15.0.4655"
+sonar-html = "3.13.0.4821"
+sonar-xml = "2.10.0.4108"
+sonar-omnisharp = "1.18.0.83276"
+sonar-text = "2.8.0.1635"
+sonar-iac = "1.26.0.8471"
+sonar-cpp = "6.52.0.66181"
```

Comment:
Small detail: for commodity, as this block is what probably changes often, I would move it closer to the top